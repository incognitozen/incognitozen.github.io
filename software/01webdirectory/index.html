<!doctype html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  
  <meta name="generator" content="Hugo 0.98.0" />

  
  <meta name="description" content="Software Alternatives,Reviews and Comparisions.">
  

  
  <link rel="apple-touch-icon" sizes="180x180" href="https://incognitozen.github.io/apple-touch-icon.png">

  
  <link rel="icon" type="image/png" sizes="32x32" href="https://incognitozen.github.io/favicon-32x32.png">

  
  <link rel="icon" type="image/png" sizes="16x16" href="https://incognitozen.github.io/favicon-16x16.png">

  
  <link rel="manifest" href="https://incognitozen.github.io/site.webmanifest">

  
  <link rel="mask-icon" href="https://incognitozen.github.io/safari-pinned-tab.svg" color="#5bbad5">
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css">
  <meta name="msapplication-TileColor" content="#da532c">
  <meta name="theme-color" content="#ffffff">
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" integrity="sha384-zCbKRCUGaJDkqS1kPbPd7TveP5iyJE0EjAuZQTgFLD2ylzuqKfdKlfG/eSrtxUkn" crossorigin="anonymous">
  
  <link rel="stylesheet" href="https://incognitozen.github.io/css/style.css" />
  
  
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-162292742-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-162292742-1');
</script>
  <script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-0342123695656251"
     crossorigin="anonymous"></script>
  
  <title>01WebDirectory | Discover softwares, their alternatives, reviews, news, questions and comparisons.</title>
  

  <style>
body {
  min-width: 300px;
  font-size: 0.875rem !important;
}

.custom-navbar {
  margin-bottom: 1em;
  height: 60px;
}

.custom-navbar a {
  display: inline-block; 
  padding: 18px 0;
  margin-right: 1em; 
  font-size: 16px;
  vertical-align: middle;
}

.custom-navbar a:hover,
.custom-navbar a:focus {
  text-decoration: none; 
}

@media print {
  .custom-navbar {
    display: none;
  }
}

article {
  padding-bottom: 1em;
}

img {
  max-width: 100%;
}


body {
  background-color: #f8f9fa;
}



body {
  color: #212529;
}



a {
  color: #007bff;
}



a:hover,
a:focus {
  color: #0056b3;
}



.custom-navbar {
  background-color: #f8f9fe;
}



.custom-navbar a {
  color: #000;
}



.custom-navbar a:hover,
.custom-navbar a:focus {
  color: #007bff;
}



.container {
  max-width: 800px;
}



pre {
  display: block;
  padding: 9.5px;
  word-break: break-all;
  word-wrap: break-word;
  background-color: #f5f5f5;
  border: 1px solid #ccc;
  border-radius: 4px;
}

pre code {
  padding: 0;
  font-size: inherit;
  color: inherit; 
  white-space: pre-wrap;
  background-color: transparent;
  border: none;
  border-radius: 0;
}

code {
  padding: 3px 4px;
  color: inherit; 
  background-color: #dde0f4;
  border: 1px solid #ccc;
  border-radius: 4px;
  font-size: .9em;
}



blockquote,
.blockquote {
  padding: 10px 20px;
  margin: 0 0 20px;
  font-size: 1em;
  border-left: 5px solid #6c757d;
}

</style>
</head>

<body>
  <nav class="custom-navbar ">
    <a class="navbar-brand pl-5" href="/">
      <img src="/images/logo.png" width="50" height="50" class="d-inline-block align-top" alt="">
    </a>
    
    <a href="/verified/">Verified</a>
    
</nav>
  
  <div class="container">
    <article>
      <div class="row">
  <div class="col-sm-12">
    
      <div class="card mb-3">
        
      <div class="card-body">
        
        <div class="media">
          
          <div class="my-12">
          <div class="media-body">
            <h1 class="mt-0"><a href="https://incognitozen.github.io/software/01webdirectory/">01WebDirectory</a><dull><a href="https://www.01webdirectory.com/" target="_blank"><i class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull></h1>
            <div class="mt-2"><p>A general web directory that lists business and informative websites under various categories for&hellip;</p>
</div>
          </div>
        </div>
      </div>
    </div>
    </div>
    <h3>11 Alternatives To 01WebDirectory</h3>
    
    
       
      
      <div class="card mb-3">
        
        <div class="card-body">
           
          <div class="media">
            
            <div class="media-body"> 
              <h1 class="mt-0"><a href="https://incognitozen.github.io/software/a1webdirectory/">A1WebDirectory</a><dull><a href="http://www.a1webdirectory.org" target="_blank"><i class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull></h1>
              <div class="mt-2">Web directory that provides website listings with short descriptions for reference.</div>
            </div>
          </div>
        </div>
      </div>
      
    
       
      
      <div class="card mb-3">
        
        <div class="card-body">
           
          <div class="media">
            
            <div class="media-body"> 
              <h1 class="mt-0"><a href="https://incognitozen.github.io/software/adopt-a-contractor/">Adopt-a-Contractor</a><dull><a href="https://adopt-a-contractor.com/" target="_blank"><i class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull></h1>
              <div class="mt-2">Adopt A Contractor is a website which helps you to get connected with Verified and Top-Rated Home improvement Pros for free.</div>
            </div>
          </div>
        </div>
      </div>
      
    
       
      
      <div class="card mb-3">
        
        <div class="card-body">
           
          <div class="media">
            
            <div class="media-body"> 
              <h1 class="mt-0"><a href="https://incognitozen.github.io/software/best_of_the_web_directory/">Best of the Web Directory</a><dull><a href="https://botw.org/" target="_blank"><i class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull></h1>
              <div class="mt-2">Editors hand-pick every website and blog, making listings consistent, relevant, and unique.</div>
            </div>
          </div>
        </div>
      </div>
      
    
       
      
      <div class="card mb-3">
        
        <div class="card-body">
           
          <div class="media">
            
            <div class="media-body"> 
              <h1 class="mt-0"><a href="https://incognitozen.github.io/software/dirpopulus/">DIRpopulus</a><dull><a href="http://ww1.dirpopulus.org" target="_blank"><i class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull></h1>
              <div class="mt-2">Web directory edited by people.</div>
            </div>
          </div>
        </div>
      </div>
      
    
      
    
      
    
       
      
      <div class="card mb-3">
        
        <div class="card-body">
           
          <div class="media">
            
            <div class="media-body"> 
              <h1 class="mt-0"><a href="https://incognitozen.github.io/software/jwtgov_web_directory/">JWTGOV Web Directory</a><dull><a href="http://www.jwtgov.com" target="_blank"><i class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull></h1>
              <div class="mt-2">Free complex catalogue of web resources.</div>
            </div>
          </div>
        </div>
      </div>
      
    
      
    
       
      
      <div class="card mb-3">
        
        <div class="card-body">
           
          <div class="media">
             
            
            <img class="mr-3 align-self-center" src="/images/2020/03/totalfinder_hue0550a42aab09397c31d1cd11cc3a07a_2388_64x64_fill_q75_box_center.jpg" alt="images/2020/03/totalfinder.jpg}}">
            
            
            <div class="media-body"> 
              <h1 class="mt-0"><a href="https://incognitozen.github.io/software/totalfinder/">TotalFinder</a><dull><a href="https://totalfinder.binaryage.com/" target="_blank"><i class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull></h1>
              <div class="mt-2">TotalFinder is a plugin for Finder.app which adds tabs like in Chrome browser, dual panels similar to TotalCommander and more tweaks.</div>
            </div>
          </div>
        </div>
      </div>
      
    
       
      
      <div class="card mb-3">
        
        <div class="card-body">
           
          <div class="media">
             
            
            <img class="mr-3 align-self-center" src="/images/2020/04/Votable_hu4da988159d6161464cbe185325470a55_3158_64x64_fill_q75_box_center_3.png" alt="images/2020/04/Votable.png}}">
            
            
            <div class="media-body"> 
              <h1 class="mt-0"><a href="https://incognitozen.github.io/software/votable/">Votable</a><dull><a href="https://votable.net" target="_blank"><i class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull></h1>
              <div class="mt-2">The largest human-edited web directory.</div>
            </div>
          </div>
        </div>
      </div>
      
    
       
      
      <div class="card mb-3">
        
        <div class="card-body">
           
          <div class="media">
             
            
            <img class="mr-3 align-self-center" src="/images/2020/04/W3DIR_hu83c07d89bc007c66dd2d8050d85e0d84_4261_64x64_fill_q75_box_center_3.png" alt="images/2020/04/W3DIR.png}}">
            
            
            <div class="media-body"> 
              <h1 class="mt-0"><a href="https://incognitozen.github.io/software/w3dir/">W3DIR</a><dull><a href="https://w3dir.com/en" target="_blank"><i class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull></h1>
              <div class="mt-2">W3DIR is a web directory powered by humans.</div>
            </div>
          </div>
        </div>
      </div>
      
    
  
 </div>
</div>

    </article>
    <div class="row ">
<div class="col-3"></div>
<div class="col-6">
<script async src="https://cse.google.com/cse.js?cx=7849a5f521f9c7798"></script>
<div class="gcse-search"></div>
</div>
<div class="col-3"></div>
</div>
  </div>
</body>


</html>
