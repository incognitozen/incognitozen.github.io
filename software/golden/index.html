<!doctype html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css">
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" integrity="sha384-zCbKRCUGaJDkqS1kPbPd7TveP5iyJE0EjAuZQTgFLD2ylzuqKfdKlfG/eSrtxUkn" crossorigin="anonymous"><link rel="stylesheet" href="https://hackerspad.net/css/style.css" />
  <title>[aard_dictionary alchetron ancient_history_encyclopedia everipedia evopedia fandom infogalactic learn_anything learnawesome libregamewiki metacademy miraheze moodle oppia scholarpedia the_atlan_data_wiki wikigalaxy wikiindex wikipedia] Alternatives To Golden | Hackers Pad</title>
  <meta name="description" content="Mapping all human knowledge"/>
  

  <style>
body {
  min-width: 300px;
  font-size: 0.875rem !important;
}

.custom-navbar {
  margin-bottom: 1em;
  height: 60px;
}

.custom-navbar a {
  display: inline-block; 
  padding: 18px 0;
  margin-right: 1em; 
  font-size: 16px;
  vertical-align: middle;
}

.custom-navbar a:hover,
.custom-navbar a:focus {
  text-decoration: none; 
}

@media print {
  .custom-navbar {
    display: none;
  }
}

article {
  padding-bottom: 1em;
}

img {
  max-width: 100%;
}


body {
  background-color: #f8f9fa;
}



body {
  color: #212529;
}



a {
  color: #007bff;
}



a:hover,
a:focus {
  color: #0056b3;
}



.custom-navbar {
  background-color: #f8f9fe;
}



.custom-navbar a {
  color: #000;
}



.custom-navbar a:hover,
.custom-navbar a:focus {
  color: #007bff;
}



.container {
  max-width: 800px;
}



pre {
  display: block;
  padding: 9.5px;
  word-break: break-all;
  word-wrap: break-word;
  background-color: #f5f5f5;
  border: 1px solid #ccc;
  border-radius: 4px;
}

pre code {
  padding: 0;
  font-size: inherit;
  color: inherit; 
  white-space: pre-wrap;
  background-color: transparent;
  border: none;
  border-radius: 0;
}

code {
  padding: 3px 4px;
  color: inherit; 
  background-color: #dde0f4;
  border: 1px solid #ccc;
  border-radius: 4px;
  font-size: .9em;
}



blockquote,
.blockquote {
  padding: 10px 20px;
  margin: 0 0 20px;
  font-size: 1em;
  border-left: 5px solid #6c757d;
}

</style>
</head>

<body>
  <nav class="custom-navbar ">
    <a class="navbar-brand pl-5" href="/">
      <img src="/images/logo.png" width="50" height="50" class="d-inline-block align-top" alt="">
    </a>
    
    <a href="/verified/">Verified</a>
    
</nav>
  
  <div class="container">
    <article>
      <div class="row">
  <div class="col-sm-12">
    
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            <div class="my-12">
              <div class="media-body">
                <h1 class="mt-0"><a href="https://hackerspad.net/software/golden/">Golden</a>
                  <dull><a href="https://golden.com" target="_blank"><i
                        class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
                </h1>
                <div class="mt-2"><p>Mapping all human knowledge</p>
</div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <h3>19 Alternatives To Golden</h3>
      
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
          
            
          
        
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/aard_dictionary/">Aard Dictionary</a>
                <dull><a href="http://aarddict.org" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">Dictionary and offline Wikipedia reader.</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            
            <img class="mr-3 align-self-center" src="/images/2020/04/Alchetron_hua4c64ed2fe981a19f2f172ebe2d786a1_60202_64x64_fill_q75_box_center_3.png"
              alt="images/2020/04/Alchetron.png}}">
            
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/alchetron/">Alchetron</a>
                <dull><a href="https://alchetron.com/" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">Visit Alchetron it is a free social encyclopedia for Internet users where you can search &amp; share thousands of organized topics which can be anything and everything !!</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/ancient_history_encyclopedia/">Ancient History Encyclopedia</a>
                <dull><a href="https://www.ancient.eu/" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">Ancient History Encyclopedia is a nonprofit company that aims to provide superior ancient history&hellip;</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            
            <img class="mr-3 align-self-center" src="/images/2020/04/Everipedia_hu8a39b5ddc23e1ce2530bace0108fd912_3161_64x64_fill_q75_box_center_3.png"
              alt="images/2020/04/Everipedia.png}}">
            
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/everipedia/">Everipedia</a>
                <dull><a href="https://everipedia.org/" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">The world&rsquo;s largest english encyclopedia</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/evopedia/">Evopedia</a>
                <dull><a href="http://evopedia.info" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">Evopedia is an offline Wikipedia viewer. Features: no Internet connection needed</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            
            <img class="mr-3 align-self-center" src="/images/2020/04/Fandom_huae0182c4e559f1ccf487ed0b10b30a64_44048_64x64_fill_q75_box_center_3.png"
              alt="images/2020/04/Fandom.png}}">
            
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/fandom/">Fandom</a>
                <dull><a href="http://www.wikia.com" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">Anyone can create a wiki!</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/infogalactic/">Infogalactic</a>
                <dull><a href="https://infogalactic.com/info/Main_Page" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">The planetary knowledge core</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/learn_anything/">Learn Anything</a>
                <dull><a href="https://learn-anything.xyz" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">Search Interactive Maps to Learn Anything</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/learnawesome/">LearnAwesome</a>
                <dull><a href="https://learnawesome.org" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">Humanity&rsquo;s universal learning map</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            
            <img class="mr-3 align-self-center" src="/images/2020/04/Libregamewiki_hu30077d48194c2b43ff6c5f43d4c85c54_44032_64x64_fill_q75_box_center_3.png"
              alt="images/2020/04/Libregamewiki.png}}">
            
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/libregamewiki/">Libregamewiki</a>
                <dull><a href="https://libregamewiki.org/Main_Page" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">the free gaming encyclopedia.</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/metacademy/">Metacademy</a>
                <dull><a href="https://metacademy.org" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">Metacademy - a free open source platform for efficient, personalized learning.</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/miraheze/">Miraheze</a>
                <dull><a href="https://miraheze.org" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">Miraheze is a wiki farm (hosts wikis) for free and with no ads, it also provides custom domains&hellip;</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            
            <img class="mr-3 align-self-center" src="/images/2020/04/Moodle_hud46d0fe59d222667cdb783efc8c07ad6_4819_64x64_fill_q75_box_center_3.png"
              alt="images/2020/04/Moodle.png}}">
            
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/moodle/">Moodle</a>
                <dull><a href="https://moodle.org:443/" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">Moodle - Open-source learning platform | Moodle.org</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/oppia/">Oppia</a>
                <dull><a href="https://www.oppia.org/splash" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">Oppia is a free, open-source learning platform.</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/scholarpedia/">Scholarpedia</a>
                <dull><a href="http://www.scholarpedia.org" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">Scholarpedia is a wiki-based encyclopedia that features the English language articles and helpful materials only.</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            
            <img class="mr-3 align-self-center" src="/images/2020/04/The-Atlan-Data-Wiki_huc2ef925d043097fd118e6658149191e3_885_64x64_fill_q75_box_center_3.png"
              alt="images/2020/04/The-Atlan-Data-Wiki.png}}">
            
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/the_atlan_data_wiki/">The Atlan Data Wiki</a>
                <dull><a href="https://wiki.atlan.com" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">A free, fun and helpful encyclopedia for the data universe</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/wikigalaxy/">WikiGalaxy</a>
                <dull><a href="http://wiki.polyfra.me" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">Explore wikipedia in 3d</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/wikiindex/">WikiIndex</a>
                <dull><a href="https://wikiindex.org/Welcome" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">Welcome to WikiIndex, a wiki of all wiki, wiki people, wiki software, wiki farms, and wiki ideas, a&hellip;</div>
            </div>
          </div>
        </div>
      </div>
      
      
      <div class="card mb-3">
        
        <div class="card-body">
          
          <div class="media">
            
            <div class="media-body">
              <h1 class="mt-0"><a href="https://hackerspad.net/software/wikipedia/">Wikipedia</a>
                <dull><a href="https://www.wikipedia.org" target="_blank"><i
                      class="pl-2 bi bi-box-arrow-in-up-right"></i></a></i></dull>
              </h1>
              <div class="mt-2">Wikipedia is a free online encyclopedia, created and edited by volunteers around the world and hosted by the Wikimedia Foundation.</div>
            </div>
          </div>
        </div>
      </div>
      
    </div>
  </div>
  
    </article>
    <div class="row ">
<div class="col-3"></div>
<div class="col-6">
<script async src="https://cse.google.com/cse.js?cx=7849a5f521f9c7798"></script>
<div class="gcse-search"></div>
</div>
<div class="col-3"></div>
</div>


<script async src="https://www.googletagmanager.com/gtag/js?id=UA-162292742-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-162292742-1');
</script>
<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-0342123695656251"
     crossorigin="anonymous"></script>

  </div>
</body>


</html>
